app/namespace.go

```diff
@@ -82,6 +124,43 @@ func GetNamespaceClient(ctx *cli.Context) (*NamespaceClient, error) {
 	return NewNamespaceClient(ct, conn), nil
 }
 
+func (c *NamespaceClient) getExistingExportSink(ctx *cli.Context, nameSpaceName, sinkName string) (*sink.ExportSink, error) {
+	getRequest := &namespaceservice.GetExportSinkRequest{
+		Namespace: nameSpaceName,
+		SinkName:  sinkName,
+	}
+
+	getResp, err := c.client.GetExportSink(c.ctx, getRequest)
+	if err != nil {
+		return nil, err
+	}
+	return getResp.Sink, nil
+}
+
+func (c *NamespaceClient) getExportSinkResourceVersionWithSink(ctx *cli.Context, sink *sink.ExportSink, nameSpaceName, sinkName string) (string, error) {
```

Comment:
[Nit] this signature doesn't need to return an error

Reply:
also why does this need namespaceName and sinkName as an argument?

Reply:
remove error and sinkName. I've done some refactoring before and forgot to remove those unneeded input parameter.